### ISSUE TITLE
build instructions

### ISSUE THREAD

**Issue by tuukka** [amarchen/Wikipedia#2]
It would be very helpful for getting started to have instructions for building the project on the command line as well as importing and building in Qt Creator.

**Comment by amarchen** [comment]
I agree, unfortunately, I don't know how to do a command line build at the moment. And to my understanding there is no simple setup in general at the moment. If somebody knows how to do it, sure, pull request would be very welcome.

**Comment by tuukka** [comment]
Figured this out in pull request #3, please test.
